src/IotHub/IotHub/Models/PSIotHubProperties.cs

```diff
@@ -69,11 +69,6 @@ public class PSIotHubProperties
         [JsonProperty(PropertyName = "comments")]
         public string Comments { get; set; }
 
-        /// <summary>
-        /// </summary>
-        [JsonProperty(PropertyName = "operationsMonitoringProperties")]
-        public PSOperationsMonitoringProperties OperationsMonitoringProperties { get; set; }
```

Comment:
@anusapan this property should not be removed

Reply:
changes reverted